wormhole/file_transport.go

```diff
@@ -308,7 +315,7 @@ func (t *fileTransport) connectToSingleHost(ctx context.Context, addr string, su
 func (t *fileTransport) directRecvHandshake(ctx context.Context, conn net.Conn, successChan chan net.Conn, failChan chan string) {
 	expectHeader := t.senderHandshakeHeader()
 
-	addr := t.relayURL.Addr()
+	addr := t.relayURL.Hostname()
```

Comment:
It looks like the `addr` gets written to the `failChan` and this function gets called from `file_transport.go:connectDirect()`. The `failChan` value is not being used as of now. I was just wondering if we want to have it in the form of "hostname:port" just for consistency. In `connectToSingleHost`, we are writing `addr` which is in the "host:port" form. But here we don't.

Reply:
Since this is for error reporting, I'm in favor of not printing the port. (If we print the port, what happens if it is not present? Where do we stop, why shouldn't we also print the full path and username and password or something?) I'll correct it in `connectToSingleHost` then.